ee/ucp/kubernetes/storage/use-csi.md

```diff
@@ -39,6 +38,20 @@ on the cluster.
 6. Perform static or dynamic provisioning of PVs using the CSI plugin as the provisioner. For details on how 
 to provision volumes for your specific storage provider, refer to the storage provider’s user manual.
 
+### Certified CSI drivers
+The following table lists the UCP certified CSI drivers.
+
+| Partner name | Kubernetes on Docker Enterprise 3.0 |
+|--------------|-------------------------------------|
+| NetApp       | Certified (Trident - CSI)           |
+| EMC/Dell     | Certified (VxFlexOS CSI driver)     |
+| VMware       | Certified (CSI)                     |
+| Portworx     | Certified (CSI)                     |
+| Nexenta      | Certified (CSI)                     |
+| Blockbridge  | Certified (CSI)                     |
+| Storidge     | Certified (CSI)                     |
```

Comment:
Storidge? Is that a real partner? Havent heard that one before.

Reply:
It's in the list here: https://docker.atlassian.net/wiki/spaces/SDX/pages/61833302/Storage+Partners

Reply:
Yup, they're a pretty new startup. I've been working with them since before they had a release product. Their tech is seriously solid.